src/style.css

```diff
@@ -0,0 +1,95 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 11px;
+  line-height: 15px;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+  display: flex;
+}
+
+.card-item {
+  width: 200px;
+  margin: 0 auto;
+}
+
+.product_img {
+  margin: 30px 20px 45px;
```

Comment:
don't use margins/paddings to center elements inside product card. Use flex properties on card-item

Reply:
display: block;
margin: 0 auto
+ https://jedicss.com/jar-and-block/